### DISCUSSION TITLE
Errors building daisyUI locally

### DISCUSSION THREAD

**Discussion by sami-29** [saadeghi/daisyui#1607] [General]
I want to start contributing to the development of daisyUI but I've run into errors that I can't tell how to resolve.

Here is the logs file:
[2023-02-16T18_46_50_947Z-debug-0.log](https://github.com/saadeghi/daisyui/files/10759479/2023-02-16T18_46_50_947Z-debug-0.log)

**Comment by saadeghi** [comment]
This doesn't seem to be an issue from daisyUI.  
But a problem from your environment while running Node.js

Take a look at this: https://stackoverflow.com/questions/57879150/how-can-i-solve-error-gypgyp-errerr-find-vsfind-vs-msvs-version-not-set-from-c

Let me know if I can help.
